hail/src/main/scala/is/hail/lir/X.scala

```diff
@@ -65,8 +65,6 @@ class Classx[C](val name: String, val superName: String, var sourceFile: Option[
       SimplifyControl(m)
     }
 
-    val writeIRs = HailContext.isInitialized && HailContext.getFlag("write_ir_files") != null
```

Comment:
This is  where the HailContext leaked in asm4s.

Reply:
fix looks good.